playwright/utils/fetchOmnichannelConfig.ts

```diff
@@ -8,6 +8,8 @@ const fetchOmnichannelConfig = (scenario = ""): object => {
     omnichannelConfig.orgId = testConfig["DefaultSettings"].orgId;
     omnichannelConfig.orgUrl = testConfig["DefaultSettings"].orgUrl;
     omnichannelConfig.widgetId = testConfig["DefaultSettings"].widgetId;
+    omnichannelConfig.token = testConfig["DefaultSettings"].token;

```

Comment:
Why is token needed? We ideally don't want to hardcode this value

Reply:
Removed.